syntax/forth.vim

```diff
@@ -195,11 +195,15 @@ syn match forthFloat '\<-\=\d*[.]\=\d\+[DdEe][-+]\d\+\>'
 syn region forthComment start='0 \[if\]' end='\[endif\]' end='\[then\]' contains=forthTodo
 
 " Strings
-syn region forthString start=+\.*\"+ end=+"+ end=+$+ contains=@Spell
+syn region forthString start=+\.*"+ end=+"+ end=+$+ contains=@Spell
```

Comment:
Do you have any idea what this is? `\"` ?

Reply:
From my perspective... I would like to either remove `\"`, or have a comment that describes where it is from.